README.md

```diff
@@ -42,45 +42,116 @@ $ sudo npm link # optional: link CLI for system-wide execution
 
 ## Usage
 
-The package contains a CLI, Docker Compose files and configuration presets.
-
-### Configuration presets
-
- - Local - standalone masternode for local development
- - Evonet - masternode with Evonet configuration
- - Testnet - masternode with testnet configuration
+The package contains a CLI, Docker Compose files and system configurations.
 
 ### CLI
 
 The CLI can be used to perform routine tasks. Invoke the CLI with `mn` if linked during installation, or with `node bin/mn` if not linked. To list available commands, either run `mn` with no parameters or execute `mn help`. To list the help on any command just execute the command, followed by the `--help` option
 
+### Configure node
+
+The `config` command is used to describe your node configuration before starting the node. Several system configurations are provided as a starting point:
+
+ - Default - basic config for use as template
+ - Local - standalone node for local development
+ - Evonet - node with Evonet configuration
+ - Testnet - node with testnet configuration
+
+```
+USAGE
+  $ mn config
+
+OPTIONS
+  --config=config  configuration name to use
+
+DESCRIPTION
+  Shows current default configuration name
+
+COMMANDS
+  config:create  Create config
+  config:envs    Export config to envs
+  config:get     Get config option
+  config:list    Lists available configurations
+  config:remove  Remove config
+  config:reset   Reset config
+  config:select  Set config as default
+  config:set     Set config option
+  config:show    Show config options
+```
+
+To show the currently selected config:
+```bash
+$ mn config
+```
+
+To list existing configs:
+```bash
+$ mn config:list
+```
+
+To create a config based on an existing config:
+```bash
+$ mn config:create mn1 testnet
+```
+
+To select a config:
```

Comment:
To set config as a default one.


Maybe we shouldn’t just list them (they already listed above) but explain flow: you need to choose config from already existing using config:select or you can create your own with config:create. You can you one of existing as a source for you config. By default the source will be ”default”. After that you need to select your config as default with config:select